main.py

```diff
@@ -45,6 +50,14 @@
 app.mount("/static", StaticFiles(directory="static"), name="static")
 
 
+# Create a Limiter instance
+limiter = Limiter(key_func=lambda request: request.headers.get('User-Agent'))
```

Comment:
**Issue**: Using 'User-Agent' header as a key function for rate limiting can be easily bypassed and is not a reliable method of identifying unique users. <br> **Fix**: Use a more reliable key function for rate limiting, such as IP address or a user-specific token that is harder to spoof. <br> **Code Suggestion**: 
 ```
 @@ -54,1 +54,1 @@
 -limiter = Limiter(key_func=lambda request: request.headers.get('User-Agent'))
 +limiter = Limiter(key_func=get_remote_address)
 ```